README.md

```diff
@@ -1,4 +1,61 @@
-# v24-geckos-team-02
-Books Plus | Voyage-24 | https://chingu.io/
+# Books Plus
 
-Netlify | booksplus-beta | https://booksplus-beta.netlify.app/
\ No newline at end of file
+## Overview
+
+BooksPlus is an App that customizes features using the Google Books Api that searches the full text of books and magazines that Google has scanned. 
```

Comment:
"Books Plus is a Google Books clone with added features. It makes use of the Google Books API."